templates/joomla/css/custom.css

```diff
@@ -17,6 +17,18 @@
 .itemid-122 .footer {
     margin-top: 0;
 }
+
+/*error page display*/
+.error.itemid-122 #content {
```

Comment:
I did an override with a `display:none` only the itemid-122, so we're not supposed to have any issue with any other itemid.